src/network/types/msg.ts

```diff
@@ -34,3 +35,28 @@ export interface DocSigner {
 	/** account public key */
 	publicKey: Uint8Array;
 }
+
+export interface DocV2 {
+	/**
+	 * chain_id is the unique identifier of the chain this transaction targets.
+	 * It prevents signed transactions from being used on another chain by an
+	 * attacker
+	 */
+	chainId: string;
+	/**
+	 * Transaction requested Fee
+	 */
+	fee: Fee;
+	/**
+	 * Transaction memo
+	 */
+	memo?: string;
+	/**
+	 * Transactions messages
+	 */
+	messages: EncodeObject[];
+	/**
+	 * Transction auth signers
+	 */
+	signers: DocSigner[];
+}
```

Comment:
@jay-dee7 What's the exact reason for this interface definition?

Reply:
@Eengineer1 I'm just trying to phase the entire LUM-related fields. The regular `Doc` uses `Message` from lum-sdk and I just want to make it cleaner. We'll just reuse this Doc everywhere, rewrite everything with cosmjs and remove the Doc from here